content/source/docs/enterprise/api/workspaces.html.md

```diff
@@ -12,40 +12,98 @@ Workspaces represent running infrastructure managed by Terraform.
+| --------------------------------------------- | ------- | ---------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `data.type`                                   | string  |            | Must be `"workspaces"`.                                                                                                                                                                                                                 |
+| `data.attributes.name`                        | string  |            | Specifies the name of the workspace, which can only include letters, numbers, `-`, and `_`. This will be used as an identifier and must be unique in the organization.                                                                  |
+| `data.attributes.auto-apply`                  | boolean | false      | Whether to automatically apply changes when a Terraform plan is successful.                                                                                                                                                             |
+| `data.attributes.terraform-version`           | string  |            | The version of Terraform to use for this workspace.                                                                                                                                                                                     |
+| `data.attributes.migration-environment`       | string  |            | Specifies the legacy environment to use as the source of the migration in the form `organization/environment`                                                                                                                           |
+| `data.attributes.working-directory`           | string  |            | Specifies a relative path that Terraform will execute within. This defaults to the root of your repository and is typically set to a subdirectory matching the environment when multiple environments exist within the same repository. |
+| `data.attributes.vcs-repo.oauth-token-id`     | string  |            | Specifies the VCS Connection (OAuth Conection + Token) to use as identified. This ID can be obtained from the [oauth-tokens](./oauth-tokens.html) endpoint.                                                                             |
+| `data.attributes.vcs-repo.branch`             | string  | `"master"` | Specifies the repository branch that Terraform will execute from. If left null or submitted as an empty string, this defaults to the repository's default branch (e.g. `master`) .                                                      |
```

Comment:
should probably put `""` as the default, since it won't default to master if someone set the default branch to `production` or something.